### DISCUSSION TITLE
v0.96.1 introduced a breaking change?

### DISCUSSION THREAD

**Discussion by helpermethod** [jbangdev/jbang#1421] [Q&A] [answered]
When JBang `v0.96.1` improved the native image building by introducing the new `native` export option, the old way of building native images broke

<img width="778" alt="image" src="https://user-images.githubusercontent.com/1562019/182546102-0df8378c-a1e1-4d79-aea1-b2f74d792e69.png">

While this is easy to fix, it's still not a what I would expect from a patch release 😆 .

**Comment by maxandersen** [comment, marked as answer]
ouch! that is my bad :(

Totally agree it should have been a 0.97 release now i realize the --native was removed.

Can't really undo it so best I can provide is a heartfelt I'm sorry!


![](https://i.pinimg.com/736x/36/01/ef/3601ef18e723a604012ed4ddf89a810d.jpg)

**Reply by quintesse** [reply]
You're absolutely right! We'll definitely do better next time. Thanks for pointing this out to us 👍

**Reply by helpermethod** [reply]
No problem for me, just wanted to ask if this was intended or not.

I love the new native export option BTW!

**Reply by maxandersen** [reply]
just pushed release button on a patch that at least tell users about the change.

`jbang export local --native myapp.java
[jbang] [ERROR] The `-n` and `--native` flags have been removed, use `jbang export native` instead.
[jbang] Run with --verbose for more details`

**Reply by maxandersen** [reply]
https://github.com/jbangdev/jbang/releases/tag/v0.96.4
